contributing/code/core_team.rst

```diff
@@ -170,8 +153,7 @@ A pull request **can be merged** if:
 
 * It is a minor change [1]_;
 
-* Enough time was given for peer reviews (at least 2 days for "regular"
-  pull requests, and 4 days for pull requests with "a significant impact");
+* Enough time was given for peer reviews;
 
 * At least the component's **Merger** or two other Core members voted ``+1``
```

Comment:
I don't know if votes as comments still work with the gh tool. But we currently use Github Reviews (Approve and Request Changes) for voting basically. I think we should update this here.

Reply:
IIRC both counts equally.

Reply:
I would remove the "component's Merger" part here.

> At least two Core members voted +1

Reply:
updated